doc/tutorial/invoking-zeekctl.rst

```diff
@@ -1,9 +1,247 @@
+.. _ZeekControl documentation: https://github.com/zeek/zeekctl
+.. _ZeekControl cron: https://github.com/zeek/zeekctl?tab=readme-ov-file#zeekcontrol-cron-command
+
 .. _invoking-zeekctl:
 
-================
-Invoking Zeekctl
-================
+==============
+ ZeekControl
+==============
+
+ZeekControl (or ``zeekctl``) is Zeek's primary tool for cluster orchestration.
```

Comment:
I have used "Zeekctl" in the NEWS recently, but I like the idea of using ZeekControl as "the spelling" when it's not about invoking the command.